agent/src/iconfucius/cli/chat.py

```
@@ -1109,70 +1118,66 @@ def run_chat(persona_name: str, bot_name: str, verbose: bool = False,
+                    )
+                finally:
+                    set_progress_callback(None)
+                    set_status_callback(None)
+            if startup_balance_result and startup_balance_result.get("status") == "ok":
+                display_text = startup_balance_result.pop("_display", "")
+                if display_text:
+                    print(f"\n{display_text}\n")
+        except (KeyboardInterrupt, EOFError):
+            print()  # user cancelled — continue to chat
         except Exception:
             pass  # non-critical — don't block chat startup
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)